RELEASE.md

```diff
@@ -7,20 +7,26 @@ Please follow the established format:
 -->
 # Next release
 
+# Release 9.1.0
+
 ## Major features and improvements
+- Include expandAllPipelines in initial state. (#1896)
```

Comment:
This feature is not a user facing change. Can we move this to the section - ## Bug fixes and other changes